docs/source/conf.py

```diff
@@ -0,0 +1,369 @@
+# -*- coding: utf-8 -*-
+#
+# cocotb documentation build configuration file
+#
+# This file is execfile()d with the current directory set to its containing dir.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import datetime
+import os
+import subprocess
+import sys
+
+# Add in-tree extensions to path
+sys.path.insert(0, os.path.abspath("../sphinxext"))
+
+import cocotb_bus
+# from cocotb_bus._vendor.distutils_version import LooseVersion
+from looseversion import LooseVersion
+
+os.environ["SPHINX_BUILD"] = "1"
+
+# -- General configuration -----------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be extensions
+# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.doctest",
+    "sphinx.ext.todo",
+    "sphinx.ext.coverage",
+    "sphinx.ext.imgmath",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.extlinks",
+    "sphinx.ext.intersphinx",
+    "sphinxcontrib.makedomain",
+    "sphinx.ext.inheritance_diagram",
+    "sphinxcontrib.cairosvgconverter",
+    "breathe",
+    "sphinx_issues",
+    "sphinx_argparse_cli",
+    "sphinxcontrib.spelling",
+    "sphinx_tabs.tabs",
+    "sphinxcontrib.details.directive",
+]
+
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
+    "ghdl": ("https://ghdl.github.io/ghdl", None),
+    "scapy": ("https://scapy.readthedocs.io/en/latest", None),
+    "pytest": ("https://docs.pytest.org/en/latest/", None),
+    #"cocotb": ("https://docs.cocotb.org/en/latest/", None),
```

Comment:
If I keep this line, in my local test with `nox -s docs_preview` the base url for :ref:, :any: links is https://docs.cocotb.org instead of http://127.0.0.1:8000

Reply:
Let's figure this out in a later PR then.